oprf/group/group.go

```diff
@@ -0,0 +1,118 @@
+package group
```

Comment:
group package must live outside oprf group.
/circl/group
/circl/oprf

Reply:
Yes! Just what I was going to ask ;)

Reply:
Let's do this in the different PR as discussed.